tests/test_send_file.py

```diff
@@ -115,10 +118,11 @@ def test_non_ascii_name(name, ascii, utf8):
     content_disposition = rv.headers["Content-Disposition"]
     assert f"filename={ascii}" in content_disposition
 
+    """Per RFC 5987/8187, "extended" values may *not* be quoted.
+    This is in keeping with browser implementations.
+    """
     if utf8:
         assert f"filename*=UTF-8''{utf8}" in content_disposition
-    else:
```

Comment:
I'm not sure why this condition has been removed

Reply:
I think I thought it was moot at the time, but you're absolutely right - - I'll put it back.